docs/README.md

```diff
@@ -257,3 +257,56 @@ precice-aste-run --aste-config solid-config.json
 ```
 
 ASTE picks up the correct mesh files, extracts the data and passes the data to preCICE.
+
+#### Example
+
+As an example, we want to replace the `Fluid` participant of our [`perpendicular-flap`](https://precice.org/tutorials-perpendicular-flap.html) tutorial by ASTE. Therefore, we first set the export tag on the fluid participant in our configuration file (see below) and run the simulation with one of the available fluid solvers
+
+```xml
+    <participant name="Fluid">
+      ...
+      <export:vtk directory="exported-meshes" />
+    </participant>
+```
+
+After the simulation finished, the directory `exported-meshes` contains now all the necessary information in order to use ASTE in our coupled simulation. We create a new directory in the `perpendicular-flap` directory called `fluid-aste` and move the `exported-meshes` into the new directory in order to run ASTE from a separate directory. Since ASTE supports only `explicit` coupling schemes, we switch from an `implicit` coupling scheme to an `explicit` coupling scheme in the preCICE configuration file as follows:
```

Comment:
```suggestion
Once the simulation is done, the directory `exported-meshes` contains all necessary coupling data in order to use ASTE for the coupled simulation. We create a new directory in the `perpendicular-flap` directory called `fluid-aste` and move the `exported-meshes` into the new directory in order to run ASTE from a separate directory. Since ASTE supports only `explicit` coupling schemes, we switch from an `implicit` coupling scheme to an `explicit` coupling scheme in the preCICE configuration file:
```